README.md

```diff
@@ -68,14 +68,28 @@ VxP Tools and Libraries | Not published yet | WI3.4.2, WI4.2.1, WI4.2.6, WI4.4.1
 
 ### TOOLS
 
-Repository | URL/Instructions | TRISTAN Working Items | Partners                    | Status                      | License   |
----------- | ---------------- | --------------------- | --------------------------- | --------------------------- | --------- |
-IP-XACT extension for timing and power intent | Not published yet | WI5.3.3 | EPOS | Concept in development| To be determined|
-To be determined|
-GVSOC | [GVSOC (fork)](https://github.com/EEESlab/gvsoc/tree/tristan-dev) | WI5.1.5 | UNIBO | Design and Verification in progress | Apache-2.0 |
-RISC-V systemverilog-based modular model for verification  | Not published yet | WI5.2.8 | SYNT | Concept in developments | To be determined |
-ETISS | [ETISS](https://github.com/tum-ei-eda/etiss) | WI5.1.2 | TUM EDA | Design and Verification in progress | BSD 3-clause |
-cv_dv_utils | [CV_DV_UTILS](https://github.com/openhwgroup/core-v-verif) | WI3.1.5 | CEA | Initial release | Apache-2.0 |
+
+Tool | URL/Instructions | TRISTAN Working Items | Owner/Contributors  |  Users               | Description                     | License   |
+-------- | -------------- | ------------ | ---------- | -------------- | --------------------------- | --------- |
+Renode | [Renode](https://github.com/renode/renode) | 5.1.1 | Antmicro | Tampere University, NOKIA, Cargotec | Simulation Framework| |
```

Comment:
for consistency, can you pls add "W" before the working item numer?

Reply:
Done, thank you for the feedback